yt_dlp/extractor/rule34video.py

```diff
@@ -43,7 +43,7 @@ class Rule34VideoIE(InfoExtractor):
             }
         },
         {
-            'url': 'https://rule34video.com/videos/3065296/lara-in-trouble-ep-7-wildeerstudio/',
+            'url': 'https://rule34video.com/video/3065296/lara-in-trouble-ep-7-wildeerstudio/',
```

Comment:
```suggestion
            'url': 'https://rule34video.com/videos/3065296/lara-in-trouble-ep-7-wildeerstudio/',
```

Lets keep one test in old pattern